distro/otel_test.go

```diff
@@ -48,7 +48,7 @@ import (
 
 const (
 	spanName   = "test span"
-	metricName = "test instrument"
+	metricName = "test_instrument"
```

Comment:
Whiteout it the tests were failing with `invalid instrument name: test instrument: must only contain [A-Za-z0-9_.-]`